src/js/main.js

```diff
@@ -0,0 +1,338 @@
+import keys from '../data/layout.json';
+
+class Project {
+
+  constructor() {
+    this.keys = keys;
+    this.lang = 'eng';
+    this.tmpStr = '';
+    this.caps = false;
+    this.shift = false;
+    this.tmp = [];
+    this.mouseID = 0;
+  }
+
+  static createHTML() {
+
+    const container = document.createElement('div'),
+           textarea = document.createElement('textarea');
+
+    container.className = 'container';
+    textarea.className = 'textarea';
+
+    document.body.append(container);
+    document.querySelector('.container').append(textarea);
+    return true;
+  }
+
+  createKeyboard() {
+
+    const keysContainer = document.createElement('div');
+
+    keysContainer.className = 'keysContainer';
+    document.querySelector('.container').append(keysContainer);
+
+    this.keys[this.lang].forEach((item, index) => {
```

Comment:
instead of `item[0]`,`item[1]`,`item[2]` you can use destructuring:
```suggestion
    this.keys[this.lang].forEach(([buttonName, buttonValue, buttonValueAlt], index) => {
```